src/website/app/LiveProvider.js

```diff
@@ -44,7 +44,12 @@ const styles = {
       : {
           backgroundColor,
           border: `1px solid ${rgba(siteColors.slate, 0.3)}`,
-          padding: theme.space_inset_md
+          fontFamily: 'zapfino',
```

Comment:
I like the idea but I would totally vote for `Comic Sans MS` as it is likely the reigning king/queen of despised fonts.